contracts/Remittance.sol

```diff
@@ -0,0 +1,98 @@
+pragma solidity 0.5.10;
+
+import "./SafeMath.sol";
+import "./Stoppable.sol";
+
+contract Remittance is Stoppable {
+    using SafeMath for uint;
+    
+    struct Order {
+        uint deadline;
+        address emitter;
+        uint amount;
+    }
+    
+    uint constant TX_FEES = 2000;
+
+    mapping(bytes32 => Order) public orders;
+    mapping(address => uint) public fees;
+    
+    event LogNewOrder(address indexed emitter, uint amount, bytes32 hashedOTP);
```

Comment:
It would be good to have the events be a verb past tense. `LogNewOrderPlaced`, `LogWithdrawn`, `LogOrderCancelled`, `LogFeesTaken`.